polyhost/server/instance.py

```
@@ -0,0 +1,58 @@
+    authkey = authkey or protocol.load_or_create_authkey()
+    conn = None
+    try:
+        conn = Client(address, authkey=authkey)
+        if not conn.poll(timeout):
+            return False
+        msg = protocol.recv_message(conn)
+        if msg.get("method") != protocol.HELLO:
+            return False
+        ok, _ = protocol.check_hello(msg.get("params") or {})
+        return ok
+    except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)